tests/_support/AcceptanceTester.php

```diff
@@ -156,38 +174,40 @@ public function checkAllResults()
 	/**
 	 * Selects an option in a Chosen Selector based on its id
 	 *
-	 * @param   string  $selectId  The id of the <select> element
-	 * @param   string  $option    The text in the <option> to be selected in the chosen selector
+	 * @param   string $selectId The id of the <select> element
+	 * @param   string $option   The text in the <option> to be selected in the chosen selector
 	 *
 	 * @return void
 	 */
 	public function selectOptionInChosenById($selectId, $option)
 	{
 		$chosenSelectID = $selectId . '_chzn';
-		$I = $this;
+		$I              = $this;
 		$this->comment("I open the $chosenSelectID chosen selector");
```

Comment:
@pritalpatel Any reason you switch between $this and $I here (which are the same), please keep it one way 👍